Content.Server/Doors/Electronics/Systems/DoorElectronicsSystem.cs

```diff
@@ -0,0 +1,84 @@
+using System.Linq;
+using Content.Shared.Access.Components;
+using Content.Shared.DeviceNetwork.Components;
+using Content.Shared.Doors;
+using Content.Shared.Doors.Electronics;
+using Content.Shared.Interaction;
+using Content.Server.Doors.Electronics;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.Doors.Electronics;
+
+public sealed class DoorElectronicsSystem : EntitySystem
+{
+    [Dependency] private readonly UserInterfaceSystem _uiSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<DoorElectronicsComponent, DoorElectronicsUpdateConfigurationMessage>(OnChangeConfiguration);
+        SubscribeLocalEvent<DoorElectronicsComponent, DoorElectronicsRefreshUiMessage>(OnRefreshUi);
+        SubscribeLocalEvent<DoorElectronicsComponent, InteractUsingEvent>(OnInteractUsing);
+    }
+
+    public void UpdateUserInterface(EntityUid uid, DoorElectronicsComponent component)
+    {
+        List<string> accesses = new List<string>();
+
+        if (TryComp<AccessReaderComponent>(component.Owner, out var accessReader))
+        {
+            foreach (var accessList in accessReader.AccessLists)
+            {
+                var access = accessList.FirstOrDefault();
+                if (access == null)
+                    continue;
+                accesses.Add(access);
+            }
+        }
+
+        var state = new DoorElectronicsConfigurationState(accesses);
+
+        _uiSystem.TrySetUiState(uid, DoorElectronicsConfigurationUiKey.Key, state);
+    }
+
+    private void OnChangeConfiguration(EntityUid uid,
+                                       DoorElectronicsComponent component,
+                                       DoorElectronicsUpdateConfigurationMessage args)
```

Comment:
dont newline event handler params like this (there are other cases like this in the pr)